### ISSUE TITLE
CocoaLumberjack will not build for watchOS 2.0

### ISSUE THREAD

**Issue by akac** [CocoaLumberjack/CocoaLumberjack#535]
CocoaLumberjack was rejected as an implicit dependency for 'CocoaLumberjack.framework' because it doesn't contain platform 'watchsimulator' in its SUPPORTED_PLATFORMS 'iphonesimulator, iphoneos, macosx'

I'm sure I'll figure something out, but until then - it'd be nice if it would work.

**Comment by jervine10** [comment]
That's a CocoaPods issue since CocoaPods doesn't support watchOS. As a workaround you can do something like this in your Podfile:

```Ruby
post_install do |installer|
    installer.project.targets.each do |target|
        if target.name == 'Pods-CocoaLumberJack'
            config.build_settings['SUPPORTED_PLATFORMS'] = ['watchos', 'watchsimulator'];
        end
    end
end
```

This is a simple example of what you might need, but it should get you started.

**Comment by rivera-ernesto** [comment]
Thanks for the tip @jervine10 .

**Comment by akac** [comment]
We don't use CocoaPods at all. It's all manual inclusion. So I'm not sure how this helps me. 

-- 
Alex Kac CEO/Founder
Transcribed by Siri

**Comment by rivera-ernesto** [comment]
I think you should check the same `SUPPORTED_PLATFORMS` Build Setting for the CocoaLumberjack project.

**Comment by rivera-ernesto** [comment]
Also start using the newly created `swift_2.0` branch.

**Comment by bpoplauschi** [comment]
I have just released a Beta 2.1.0 version via CocoaPods including the Swift 2.0 support (also available in the latest master). Please test it and tell us if it works as expected.

**Comment by bpoplauschi** [comment]
CocoaLumberjack works fine now with WatchOS (included in Travis tests).
